### PR TITLE
common/Error

### PR DESCRIPTION
**Pull request by KarolCiesluk**
I wrote an **Error component** that's going to be used on the 6.Error Page together with the Navigation component. 
I've also added colors in **core/App/theme.js**, imported Error to **core/App/index.js** and tried to resoleve some conflicts with main branch so there's less work later :)

### CODE REVIEW

File: src/common/Error/styled.js [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+    margin: 0 auto;
+    max-width: 630px;
+    text-align: center;
+    padding: 16px;
+`;
+
+export const DangerImage = styled.img`
+    @media(max-width: ${({theme}) => theme.breakpoints.xsmall}px) {
+        width: 100px;
+    }
```
Comment (patrycja-i-lesniak) [inline src/common/Error/styled.js:14]: Why are you adding width to @media. The icon with no background is 100px. It does not need to be scaled on smaller screens.

File: src/common/Error/styled.js [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+export const Title = styled.h1`
+    margin: 24px 0;
+    font-size: 36px;
+    line-height: 120%;
+
+    @media(max-width: ${({theme}) => theme.breakpoints.xsmall}px) {
+        font-size: 20px;
+        margin: 16px 0;
+    }
```
Comment (patrycja-i-lesniak) [inline src/common/Error/styled.js:25]: If you make changes at the beginning of the code,  three lines of code will be enough here. Without @media.

font-weight: 600; font-size: 36px; line-height: 1.2;

File: src/common/Error/styled.js [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+export const Subtitle = styled.h2`
+    margin: 0 auto;
+    max-width: 450px;
+    font-weight: 500;
+    font-size: 22px;
+    line-height: 130%;
+
+    @media(max-width: ${({theme}) => theme.breakpoints.xsmall}px) {
+        font-size: 14px;
+    }
```
Comment (patrycja-i-lesniak) [inline src/common/Error/styled.js:37]: Five lines of code are enough here. And again without @media.
`max-width: 427px;
    font-weight: 500;
    margin: auto;
    font-size: 22px;
    line-height: 1.3;`

File: src/common/Error/styled.js [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+    font-size: 14px;
+    font-weight: 700;
+    line-height: 19px;
+    font-family: 'Open Sans', sans-serif;
+    text-decoration: none;
+
+    @media(max-width: ${({theme}) => theme.breakpoints.xsmall}px) {
+        padding: 12px 16px;
+        font-size: 12px;
+    }
```
Comment (patrycja-i-lesniak) [inline src/common/Error/styled.js:56]: @media again :)

**Review by patrycja-i-lesniak — CHANGES_REQUESTED** [review]
Well done. The component looks good, except that it doesn't display in the middle of the page. However, a small change in the code will make it much shorter. Test it, maybe you will like my solution.

File: src/common/Error/styled.js [hunk 5]
```diff
@@ -0,0 +1,57 @@
+import styled from "styled-components";
+import { NavLink } from "react-router-dom";
+
+export const Wrapper = styled.div`
+    margin: 0 auto;
+    max-width: 630px;
+    text-align: center;
+    padding: 16px;
```
[the remark this answers is not in the document]
Comment (WojciechBylica) [reply src/common/Error/styled.js:8]: maybe it would help us with this position of div:
margin-top: 10vw;

[the remark this answers is not in the document]
Comment (KarolCiesluk) [reply src/common/Error/styled.js:8]: Yeah I will add the margin! Patrycja, your way works great but let's just keep positioning as simple as possible (and less controversial :P) at least for now :)

File: src/common/Error/styled.js (hunk 1 above)
Comment (KarolCiesluk) [reply src/common/Error/styled.js:14]: will fix it :)

**Review by patrycja-i-lesniak — APPROVED** [review]

**Pull request merged**